recipes/notolog/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "notolog" %}
+{% set version = "0.9.1b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/notolog-{{ version }}.tar.gz
+  sha256: ab61b56d882e8b888b676a08a6fefd8dc222ed48160eb57f448dec5a8f95b82c
+
+build:
+  entry_points:
+    - notolog=notolog.main:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<4.0
+    - pip
+  run:
+    - python >=3.9,<4.0
+    - cffi ==1.16.0
+    - click ==8.1.7
+    - cryptography ==42.0.7
+    - emoji ==2.12.1
+    - iniconfig ==2.0.0
+    - Markdown ==3.6
+    - packaging ==24.0
+    - pluggy ==1.5.0
+    - pycparser ==2.22
+    - Pygments ==2.18.0
+    - PySide6 ==6.7.0
+    #- PySide6_Addons ==6.7.0
+    #- PySide6_Essentials ==6.7.0
```

Comment:
Why did you disable these?

Reply:
From PyPI:

> Please note: this wheel is an alias to other two wheels [PySide6_Essentials](https://pypi.org/project/PySide6_Essentials) and [PySide6_Addons](https://pypi.org/project/PySide6_Addons), which contains a predefined list of Qt Modules.

There is Pyside6 package for Conda-Forge

Reply:
`pip check` fails because of that difference in requirements. I removed those lines. Thank you for the review.

Reply:
`pip check` fails because of that difference in requirements. I removed those lines. Thank you for the review.